benchmarks/benchmark/tools/profile-generator/container/benchmark_serving.py

```diff
@@ -272,75 +274,126 @@ async def benchmark(
 
 
 def save_json_results(args: argparse.Namespace, benchmark_result):
-  # dimensions values are strings
-  dimensions_json = {}
-  # metrics values are numerical
-  metrics_json = {}
-
   # Setup
-  current_dt = datetime.now().strftime("%Y%m%d-%H%M%S")
-  dimensions_json["date"] = current_dt
-  dimensions_json["backend"] = args.backend
-  dimensions_json["model_id"] = args.model
-  dimensions_json["tokenizer_id"] = args.tokenizer
-  if args.additional_metadata_metrics_to_save is not None:
-    dimensions_json = {
-        **dimensions_json,
-        **json.loads(args.additional_metadata_metrics_to_save),
+  current_dt = datetime.now()
+  current_dt_proto = Timestamp()
+  current_dt_proto.FromDatetime(current_dt)
+
+  final_json = {
+    # metrics values are numerical
+    "metrics" : {
+      # Traffic
+      "num_prompts": args.num_prompts,
+      "request_rate": args.request_rate,
+      'server_metrics': {
+        **benchmark_result["metric_names"]
+      }
+    },
+    # dimensions values are strings
+    "dimensions": {
+      "date": current_dt.strftime('%Y%m%d-%H%M%S'),
+      "backend": args.backend,
+      "model_id": args.model,
+      "tokenizer_id": args.tokenizer,
+      **(json.loads(args.additional_metadata_metrics_to_save) if args.additional_metadata_metrics_to_save else {})
+    },
+    "config": {
+      "model": args.model,
+      "model_server": args.backend,
+      "start_time": current_dt_proto.ToJsonString(),
+    },
+    "summary_stats": {
+      "stats": {
+        **benchmark_result["metric_aliases"]
```

Comment:
nit `**benchmark_result.get("metric_aliases", {})` and above for metric_names so we aren't **None